### ISSUE TITLE
Executing Scoop commands show error on Windows when Username has space in it

### ISSUE THREAD

**Issue by AndreiH** [lukesampson/scoop#2823]
This was tested in multiple VMs. When username is like `C:\Users\David Petric\`, when running scoop commands I always get the following error:
```
C:\Users\David : The term 'C:\Users\David' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
again.
At line:1 char:1
+ C:\Users\David Petric\scoop\apps\scoop\current\bin\scoop.ps1 list
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\David:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

When re-creating the VM with username without empty space, Scoop does not generate this error.

**Comment by Ash258** [comment]
Do you get same error if you run `scoop.ps1 list`?

If yes, could you replace line `if($myinvocation.expectingInput) { $input | & $path  @args } else { & $path  @args }` in `$env:SCOOP\shims\scoop.ps1` with `if($myinvocation.expectingInput) { $input | & "$path"  @args } else { & "$path"  @args }`

**Comment by AndreiH** [comment]
@Ash258 Thank you for the response.
I do not get the same error when running `scoop.ps1 list`. When I run it, `scoop.ps1` file opens in Notepad. (This running from CMD)

I have now noticed that these errors are only happening when I run the Scoop commands from Git-Bash.
When running `scoop.ps1 list` command from Git-Bash, I also get:
```/c/Users/David Petric/scoop/shims/scoop.ps1: line 1: syntax error near unexpected token `!'
/c/Users/David Petric/scoop/shims/scoop.ps1: line 1: `?if (!(Test-Path Variable:PSScriptRoot)) { $PSScriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent }
```

Even after I did the modifications suggested by you in your last comment. Thank you !
